gwr/diagnostics.py

```diff
@@ -79,3 +79,11 @@ def get_CV(gwr):
     cv = np.sum(aa**2)/gwr.n
     return cv
 
+def corr(cov):
+    sd = np.diag(np.sqrt(np.diag(cov)))
+    try:
+        invsd = np.linalg.inv(sd)
+        cors = np.dot(np.dot(invsd, cov), invsd)
+    except:
```

Comment:
Also, this should probably not be a blank `except` statement... what's the anticipated error? a `numpy.linalg.LinAlgError`?

Reply:
If you raise an error with stop the routine or finish executing the code and just print the throwback?

Reply:
I don't understand the question? MaybeI can be more clear.... there're a bunch of ways the try/except could fail. If we don't tell the "except" statement which error we want to catch, it'll catch all errors, including if we give it an interrupt signal, or if the script runs out of memory... etc. 

(I don't think it'll ever fail, since sd is diagonal and positive, but, it's [recommended practice](https://docs.python.org/3/howto/doanddont.html#except))

Reply:
Sorry, I think i was unclear. When I wrote this a while ago, I think I have the try/except block in there because the intercept (an array of 1's) will always throw a singular matrix error. I don't remember why, but I guess I thought it would be a good idea to return zero's whenever there is a correlation including the intercept and that the try/except block was an easy way catch that. I don't anticipate there being any other issues. Ultimately, I think it just makes sense to not try to compute correlation coefficients with the intercept and then I should be fine to use np.corrcoef. Will try it out.

Reply:
Ooof, forgot to take out the try/except, but now I really don't expect it to fail and so its not needed.